.pre-commit-config.yaml

```diff
@@ -0,0 +1,4 @@
+- repo: "https://github.com/PyCQA/flake8"
+  rev: "3.8.3"
+  hooks:
+  - id: "flake8"
```

Comment:
I think this should say `stage: push` or similar .. I don't know about others, but I will often have commits that aren't "ready" in the sense of "production-quality code" or so but are still "a thing I want to commit".

Or, maybe I'm wrong about what a commit-hook does, but I think the above says "you can't make a commit unless flake8 passes".